docs/docs/reference/cli.md

```diff
@@ -560,14 +563,15 @@ constellation iam create gcp [flags]
   -h, --help                      help for gcp
```

Comment:
We should have a look on how we can set an individual format for the description of autogenerated help command.

Reply:
Other way round: we adapt the Cobra style